oidc_cli/token_getter.go

```diff
@@ -18,16 +20,18 @@ const (
 // GetToken gets an oidc token.
 // It handles caching with a default cache and keyring storage.
 func GetToken(ctx context.Context, clientID string, issuerURL string) (*client.Token, error) {
-	fileLock, err := pidlock.NewLock(lockFilePath)
+	f, err := os.OpenFile("/tmp/oidc-lock-time", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
still need to get rid of this timing code, wanted to verify things were  in fact quicker.